### PR TITLE
fix: lg parse template body/comments issue

### LINKED ISSUE #2152: lg inline editor swallow characters
## Describe the bug

lg inline editor swallow characters if inputing comments.

## Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## Screenshots

lg

### LINKED ISSUE #2374: lg parse nested import file performance
When a lg file import a lot lg looks like

parse become very slow. 

[EchoBot-3.zip](https://github.com/microsoft/BotFramework-Composer/files/4385430/EchoBot-3.zip)
for example this bot's entry lg file takes about 1s to parse

### PR DESCRIPTION
**Pull request by zhixzhan**
## Description

Upgrade lg parser including breaking api change, fix parse body

## Task Item

close #2152 
close #2374 
refs #2398

### CODE REVIEW

**Comment by zhixzhan** [comment]
Ready for review. Despite not all lgParser issue are fixed al, but resolved most common scenarios.

File: Composer/packages/client/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "@uifabric/styling": "^7.10.4",
     "adaptive-expressions": "^4.8.0-preview-110700",
     "axios": "^0.18.0",
-    "botbuilder-lg": "^4.8.0-preview-109324",
+    "botbuilder-lg": "^4.9.0-preview-117748",
```
Comment (a-b-r-o-w-n) [inline Composer/packages/client/package.json:40]:
```suggestion
    "botbuilder-lg": "4.9.0-preview-117748",
```

Let's lock to this version.

File: Composer/packages/lib/indexers/package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "@bfc/shared": "*",
     "@microsoft/bf-lu": "^4.9.0-preview.115707",
     "adaptive-expressions": "^4.8.0-preview-110700",
-    "botbuilder-lg": "^4.8.0-preview-109324",
+    "botbuilder-lg": "^4.9.0-preview-117748",
```
Comment (a-b-r-o-w-n) [inline Composer/packages/lib/indexers/package.json:34]:
```suggestion
    "botbuilder-lg": "4.9.0-preview-117748",
```

File: Composer/packages/tools/language-servers/language-generation/package.json [hunk 3]
```diff
@@ -16,8 +16,8 @@
   },
   "dependencies": {
     "@bfc/indexers": "*",
-    "botbuilder-lg": "^4.8.0-preview-109324",
     "adaptive-expressions": "^4.8.0-preview-110700",
+    "botbuilder-lg": "^4.9.0-preview-117748",
```
Comment (a-b-r-o-w-n) [inline Composer/packages/tools/language-servers/language-generation/package.json:20]:
```suggestion
    "botbuilder-lg": "4.9.0-preview-117748",
```

**Pull request merged**